cfme/tests/cloud/test_instance_power_control.py

```diff
@@ -192,8 +192,11 @@ def test_start(
     Metadata:
         test_flag: power_control, provision
     """
-    testing_instance.wait_for_vm_state_change(
-        desired_state=testing_instance.STATE_OFF, timeout=720, from_details=True)
+    if provider.type == 'azure':
```

Comment:
verify_vm_stopped gets state from mgmt from background like running,stopped,deleted, which doesn't reflect  vm state in web ui. and wait_for_vm_state_change simulates web ui, as we need to update page and/or refresh relationships